frank-doc-frontend/src/app/pipes/javadoc.pipe.spec.ts

```diff
@@ -12,17 +13,19 @@ describe('JavadocPipe', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [HttpClientTestingModule],
-      providers: [
+    imports: [],
+    providers: [
         AppService,
         {
-          provide: DomSanitizer,
-          useValue: {
-            bypassSecurityTrustHtml: (html: string): string => html,
-          },
+            provide: DomSanitizer,
+            useValue: {
+                bypassSecurityTrustHtml: (html: string): string => html,
+            },
         },
-      ],
-    });
+        provideHttpClient(withInterceptorsFromDi()),
+        provideHttpClientTesting(),
```

Comment:
The indentation is looking weird, But this project contains prettier, so it's good? 🤷🏻‍♂️

Reply:
Its done by the angular cli tool, so eslint & prettier didnt touch it
But I've made an issue to enforce the frontend conventions we have defined since the eslint & prettier rules arent up to date currently